### DISCUSSION TITLE
Can I ask what VBs the project uses? Are they fully local VBs?

### DISCUSSION THREAD

**Discussion by OzzyXu** [arc53/DocsGPT#534] [Q&A]
I saw two VB-related libraries: faiss and elastic. Are they the VBs the project uses? Does the project support other VBs? Thank you.

**Comment by dartpain** [comment]
You can easily integrate anything elso to store data in and to search on top of it. Just add a class in `applicaiton/vectorstore` folder

Yes currently its an option between those two.
